client/my-sites/themes/controller.js

```diff
@@ -113,17 +113,30 @@ export function loggedOut( context, next ) {
 	next();
 }
 
+function getDetailsProps( context ) {
+	const { slug } = context.params;
+	const basePath = '/themes/:slug';
+	const analyticsPageTitle = 'Themes > Details Sheet';
+
+	const runClientAnalytics = function() {
+		analytics.pageView.record( basePath, analyticsPageTitle );
+	};
+
+	return {
+		themeSlug: slug,
+		title: buildTitle(
+			i18n.translate( 'Theme Details', { textOnly: true } )
```

Comment:
> `<theme> Theme — WordPress.com` should be good.

1a8c96da306d771ede2f809aceaab1dcb2d381f8. This doesn't work yet, however, since the theme-details subtree isn't initialized yet at this stage. I'll wait and see if #3279 fixes that.